### ISSUE TITLE
[compose] NoClassDefFoundError: Failed resolution of: Landroidx/compose/runtime/MutableStateKt

### ISSUE THREAD

**Issue by vsay01** [airbnb/lottie-android#1773]
Lottie version: com.airbnb.android:lottie-compose:1.0.0-alpha06
Compose version: 1.0.0-beta02

When following the [documentation](http://airbnb.io/lottie/#/android-compose) and fetching the stable version 1.0.0-alpha06 the app will crash when using 1.0.0-beta02 versions of compose with a Class not found exception:

```
java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/runtime/MutableStateKt;
        at com.airbnb.lottie.compose.LottieAnimationState.<init>(LottieAnimationState.kt:29)
        at com.airbnb.lottie.compose.LottieAnimationStateKt.rememberLottieAnimationState(LottieAnimationState.kt:17)
        at com.airbnb.lottie.compose.LottieAnimationKt.LottieAnimation(LottieAnimation.kt:62)
```

Anyone sees this issue?

**Comment by gpeal** [comment]
@vsay01 Are you sure this is Lottie related? Have you added the actual compose libraries to your project?
https://developer.android.com/jetpack/compose/setup#configure_gradle

**Comment by vsay01** [comment]
@gpeal Yep. I followed the http://airbnb.io/lottie/#/android-compose, and see the runtime error when I call Loader compose function in the doc.

```
@Composable
fun Loader() {
    val animationSpec = remember { LottieAnimationSpec.RawRes(R.raw.loading) }
    LottieAnimation(
        animationSpec,
        modifier = Modifier.preferredSize(100.dp)
    )
}
```

Work around:
I got it work with 'com.airbnb.android:lottie-compose:1.0.0-alpha07-SNAPSHOT' and Compose version: 1.0.0-beta02.
For now I think we can close

I think we can close this if you don't see the issue. I'll re-test wants we have new version come out.
